packages/plugin-anchor/src/editor.tsx

```diff
@@ -17,6 +17,7 @@ export const AnchorEditor = (
       {focused ? (
         <EditorInput
           label="Identifier:"
+          placeholder="Gib hier einen Namen ein."
```

Comment:
This isn't very clear... My suggestion: 
"Gib den Namen für die Verlinkung ein",
"Gib die Adresse für die Verlinkung ein."
Or "Sprungmarke" instead of "Verlinkung"....

Reply:
If you want to have it shorter then remove "Gib den ... ein"

Reply:
done